python-backend/input.py

```diff
@@ -29,7 +30,7 @@ class Expression(BaseModel):
     expression: str
 
 
-def parse(data: Input) -> tuple[Callable, Number, Callable, Number, Symbol]:
+def parse(data: Input) -> tuple[Callable, Number, Callable, Number, Symbol, Number]:
```

Comment:
Consider making this a `dataclass`\-based `class` if you find yourself having to add another attribute in the future.